src/styles/blocks/card.scss

```diff
@@ -0,0 +1,116 @@
+// $btn-color: #00acdc;
+// $font-base-color: #060b35;
+
+.card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 200px;
+  height: fit-content;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: 0.3s all;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    height: 134px;
+    width: 160px;
+  }
+
+  &__infos-box {
+    margin-top: 40px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $font-base-color;
```

Comment:
Mario, I'm being extra-picky here. You could add the `transition: 0.3s all` here too, so this isn't the only styling that does not get smooth when transitioning ;D